### PR TITLE
Fix for CVMFS locations lookup

### PR DESCRIPTION
**Pull request by fstagni**
Superseeds #215 

Tests ran in https://github.com/fstagni/Pilot/actions/runs/6657162457

### CODE REVIEW

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -218,7 +254,9 @@ jobs:
         run: |
           cd Pilot
           export VO_LHCB_SW_DIR=${GITHUB_WORKSPACE}/Pilot
-          curl https://lhcbdirac.s3.cern.ch/Pilot3/LHCbPilotCommands.py -o LHCbPilotCommands.py
+          
+          curl https://gitlab.cern.ch/fstagni/LHCbPilot/-/raw/debug/LHCbPilot/LHCbPilotCommands.py -o LHCbPilotCommands.py
```
Comment (fstagni) [inline .github/workflows/integration.yml:258]: Note to myself : fix this once https://gitlab.cern.ch/lhcb-dirac/LHCbPilot/-/merge_requests/52 is merged

File: .github/workflows/integration.yml [hunk 2]
```diff
@@ -245,28 +292,62 @@ jobs:
...
+          cd Pilot
+          mkdir -p etc/grid-security/
+          echo "$HOSTCERT_BASE64" | base64 --decode > etc/grid-security/hostcert.pem
+          echo "$HOSTKEY_BASE64" | base64 --decode > etc/grid-security/hostkey.pem
+          chmod 440 etc/grid-security/hostcert.pem
+          chmod 400 etc/grid-security/hostkey.pem
+      - name: tests
+        run: |
+          cd Pilot
+          curl https://gitlab.cern.ch/fstagni/LHCbPilot/-/raw/refactor_1/LHCbPilot/LHCbPilotCommands.py -o LHCbPilotCommands.py
```
Comment (fstagni) [inline .github/workflows/integration.yml:344]: Note to myself: fix this once https://gitlab.cern.ch/lhcb-dirac/LHCbPilot/-/merge_requests/49 is merged (and add `master` branch too).

File: tests/CI/pilot_newSchema.json [hunk 3]
```diff
@@ -101,7 +106,7 @@
 			"pilotRepo": "https://github.com/does_not_matter/Pilot.git",
 			"GenericPilotGroup": "gridpp_pilot",
 			"GenericPilotDN": "VAR_USERDN_GRIDPP",
-			"RemoteLogging": "True",
+			"RemoteLogging": "False",
```
Comment (fstagni) [inline tests/CI/pilot_newSchema.json:109]: @martynia this somehow does not work. Will be tested in other PR.

Comment (martynia) [reply tests/CI/pilot_newSchema.json:109]: That's strange . It works for me locally and in  #217

**Comment by fstagni** [comment]
@atsareg @martynia this is now really ready to be merged. All integration tests succeeded: https://github.com/fstagni/Pilot/actions/runs/6705655381

File: tests/CI/pilot_newSchema.json (hunk 3 above)
Comment (martynia) [reply tests/CI/pilot_newSchema.json:109]: OK, now I see. You changed the schema and the test (obviously) failed. Now after "fixing" the test everything passes. A question remains why the original setting (with the remote logger ON) was not acceptable. This poses a potential problem when merging this and rebasing #217, where the logging is on by default.

**Review by atsareg — APPROVED** [review]

**Pull request merged**